warehouse/query-core/src/test/java/datawave/query/jexl/visitors/PullupUnexecutableNodesVisitorTest.java

```diff
@@ -1,159 +1,162 @@
 package datawave.query.jexl.visitors;
 
 import com.google.common.collect.Sets;
+import datawave.marking.MarkingFunctions;
 import datawave.query.config.ShardQueryConfiguration;
 import datawave.query.exceptions.InvalidQueryTreeException;
 import datawave.query.jexl.JexlASTHelper;
 import datawave.query.jexl.visitors.validate.ASTValidator;
 import datawave.query.util.MockMetadataHelper;
+import org.apache.accumulo.core.client.TableNotFoundException;
 import org.apache.commons.jexl2.parser.ASTJexlScript;
 import org.apache.commons.jexl2.parser.ParseException;
 import org.junit.Before;
 import org.junit.Test;
 
 import java.util.Set;
+import java.util.concurrent.ExecutionException;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
 public class PullupUnexecutableNodesVisitorTest {
-    
+
     private Set<String> indexedFields;
     private Set<String> indexOnlyFields;
     private Set<String> nonEventFields;
-    
+
     private ShardQueryConfiguration config;
     private MockMetadataHelper helper;
-    
+
     private final ASTValidator validator = new ASTValidator();
-    
+
     @Before
-    public void setup() {
+    public void setup() throws TableNotFoundException, ExecutionException, MarkingFunctions.Exception {
```

Comment:
unnecessary.  and imports unneeded after removed.

Reply:
Removed